play/src/front/Components/EmbedScreens/CamerasContainer.svelte

```diff
@@ -87,7 +87,7 @@
     });
 
     onDestroy(() => {
-        gameScene.reposition();
+        gameScene.reposition(true);
```

Comment:
Shouldn't we reposition with an animation when the container is destroyed? That would be smoother no?